lib/puppet/provider/rsync/rsync.rb

```diff
@@ -247,7 +249,7 @@ def build_command
     else
       cmd << command('rsync')
     end
-    cmd << ['-i', '-p', '-S']
+    cmd << ['-i', '-S']
```

Comment:
How about:
```
+    cmd << ['-i', '-S', '--chmod=ugo=rwX']
```

Reply:
@pillarsdotnet Soon after this goes through, we're going to make it so that `chmod` gets added as a parameter for users to be able to set what they like.

I know that, [in Slack](https://simp-project.slack.com/archives/CA4EMBXEJ/p1564596826073000) you indicated that, per the man page, that is the recommended setting and that the system `umask` would take precedence.

I agree that this makes sense, but I worry that there isn't necessarily sufficient control over the `umask` that is passed through to Puppet to allow arbitrary world access to files.

I would like to go ahead with the `o-rwx` for now and then see how it plays out in reality with the understanding that the addition of the `chmod` parameter will let people actively set it to `ugo=rwX` if necessary.

Reply:
Added [SIMP-6917](https://simp-project.atlassian.net/browse/SIMP-6917) to track the addition of the `chmod` parameter